internal/uvm/vpmem.go

```diff
@@ -150,15 +168,3 @@ func (uvm *UtilityVM) RemoveVPMEM(ctx context.Context, hostPath string) (err err
 	uvm.vpmemDevices[deviceNumber].refCount--
 	return nil
 }
-
-// PMemMaxSizeBytes returns the maximum size of a PMEM layer (LCOW)
-func (uvm *UtilityVM) PMemMaxSizeBytes() uint64 {
-	return uvm.vpmemMaxSizeBytes
-}
-
-// ExceededVPMem returns true if the addition of a new vpmem device exceeds uvm limits on vpmem
-func (uvm *UtilityVM) ExceededVPMem(fileSize int64) bool {
-	uvm.m.Lock()
-	defer uvm.m.Unlock()
-	return (uint64(fileSize) > uvm.vpmemMaxSizeBytes) || (uvm.vpmemNumDevices >= uvm.vpmemMaxCount)
-}
```

Comment:
You can get rid of all the uvm.vpmemNumDevices references as well.

Reply:
Oh good call!